packages/labs/eslint-plugin/package.json

```diff
@@ -33,7 +36,7 @@
       "clean": "if-file-deleted"
     },
     "test": {
-      "command": "node --enable-source-maps --test-reporter=dot --test test/**/*_test.js",
+      "command": "node --enable-source-maps --test-reporter=spec --test test/**/attribute-names_test.js",
```

Comment:
is narrowing the --test glob intentional?

Reply:
nope!